### ISSUE TITLE
problem rigging torso using the Spine rig on pre-comped layers

### ISSUE THREAD

**Issue by githubuser832** [RxLaboratory/DuAEF_Duik#860]
**Describe the bug**

hello, I want to report a bug (at least I think so, considering it didn't happen when I used duik 16.2.29). it happens when I tried to rig a character's torso part using the spine (probably also affects other structures as well, haven't tried them). my character's torso is a pre-comp, consisting of 3 parts, neck, torso, and hips. when I tried to rig it using the puppet tool, it seems the puppet pins are not moving together with the Controller (in this case, a center of gravity controller) despite the fact that I have parented the artwork (which is puppet tooled) to the controller. the artwork will appear to have some sort of "lines" around it, and after checking it, it was actually the puppet pins not moving at all eventhough the artwork is moved. the funny thing is, when I don't parent the pre-comped artwork to ANY controller, the artwork appears just fine, but for all I know it's not recommended as it can cause problems down the road. the problem didn't occur if the artwork is not pre-comped.
**To Reproduce**
1. pre-comp the layers ( in my case, 3 torso parts as one)
2. create spine structures (i used 3 spines)
3. position structure as needed
2. set puppet pins on the pre-comped artwork
3. create bones from the puppet pins
4. create controller for CoG ( center of gravity), hips, chest, and mid
5. parent the bones to the structures
6. parent the hips, chest, and mid to CoG
7. try to move the CoG controller

**Error description, if any**
File: 
Line number:
Error message: 

**Expected behavior**
there shouldn't be any kind of "lines" around the puppet tooled artwork (the puppet pins should move together with the artwork where the pins are). this problem didn't occur if I tried on artworks that are not pre-comped. also didn't happen back when when I was using duik 16.2.29, even when tried to pre-comped artwork

**Screenshots**
<img width="792" alt="Screen Shot 2021-09-16 at 15 07 46" src="https://user-images.githubusercontent.com/85854900/133575178-e571821c-0bcb-4c3e-8641-415d32e9bf5a.png">

**System (please complete the following information):**
 - OS: macOS big sur 11.5.2
 - Version of After Effects: 18.4 and 18.4.1
 - Version of Duik: 16.2.30

**Additional context**
my artwork appears to be "split" when the controller is moved. the puppet pins are not moving at all from the points of origin

**Comment by Nico-Duduf** [comment]
That looks like a "Collapse transformation/Continuous rasterization" issue and/or parenting the precomped artwork to something

Can you check if:
- the precomp has been parented to anything (it should not, you should be able to control it with only the pins/bones and no parenting on the layer itself)
- if the "collapse transformation" option is checked on the precomp, and try with it unchecked

You can also just drop your aep here so I can have a look ;)

Thanks!

**Comment by githubuser832** [comment]
hello, thanks for your reply. 
from what I learned in character rigging course, it was stated that I should parent my torso artwork to the COG controller. I do notice that if I don't parent my artwork to anything, the artwork and pins will follow the controller just fine with no split artwork at all. 
it seems like your second recommendation works, though! now my artwork is no longer splitting when moved, even when I parented it to the controller.
 Thank you for your help and sorry for reporting it as a bug!
